### PR TITLE
Lesson41 spring mvc controller

### PR DESCRIPTION
**Pull request by nait13**

### CODE REVIEW

File: src/main/java/org/lesson41/controller/ServletController.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+import org.lesson41.DTO.User;
+import org.lesson41.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+public class ServletController {
```
Comment (DHrytsuk) [inline src/main/java/org/lesson41/controller/ServletController.java]:
```suggestion
public class UserController {
```

File: src/main/java/org/lesson41/controller/ServletController.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    @Autowired
+    private UserService userService;
+
+    @GetMapping("/get")
+    public ModelAndView getUserForm(@RequestParam(value = "id", required = false) String id) {
```
Comment (DHrytsuk) [inline src/main/java/org/lesson41/controller/ServletController.java]: Do you need value?

File: src/main/java/org/lesson41/controller/ServletController.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        ModelAndView mav = new ModelAndView();
+        if (id == null || id.isEmpty()) {
+            mav.setViewName("getUserForm");
```
Comment (DHrytsuk) [inline src/main/java/org/lesson41/controller/ServletController.java]: I don't see pages and spring config

File: src/main/java/org/lesson41/controller/ServletController.java (hunk 2 above)
Comment (DHrytsuk) [reply src/main/java/org/lesson41/controller/ServletController.java]: Also, why it's String, not int?

File: src/main/java/org/lesson41/DTO/User.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
+package org.lesson41.DTO;
+
+public class User {
+    private int id;
+    private String name;
+    private String login;
+
+    public User(int id, String name, String login) {
```
Comment (DHrytsuk) [inline src/main/java/org/lesson41/DTO/User.java]: Do we need this constructor?

File: src/main/java/org/lesson41/controller/ServletController.java (hunk 2 above)
Comment (nait13) [reply src/main/java/org/lesson41/controller/ServletController.java]: Поменял String на Integer fix

Comment (nait13) [reply src/main/java/org/lesson41/controller/ServletController.java]: Value или name это же  Alias ?

File: src/main/java/org/lesson41/DTO/User.java (hunk 4 above)
Comment (nait13) [reply src/main/java/org/lesson41/DTO/User.java]: Fix

File: src/main/java/org/lesson41/controller/ServletController.java (hunk 2 above)
Comment (DHrytsuk) [reply src/main/java/org/lesson41/controller/ServletController.java]: Yes

Comment (DHrytsuk) [reply src/main/java/org/lesson41/controller/ServletController.java]: > Do you need value?
?

Comment (nait13) [reply src/main/java/org/lesson41/controller/ServletController.java]: Да, у меня без него не отрабатывает @RequestParam

Comment (DHrytsuk) [reply src/main/java/org/lesson41/controller/ServletController.java]: Let's return to this on the lesson

Comment (nait13) [reply src/main/java/org/lesson41/controller/ServletController.java]: Approved?)

**Review by DHrytsuk — APPROVED** [review]

**Pull request closed without merging**